src/carriers/zfp_portmonitor/zfpPortmonitor.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2016 iCub Facility
+ * Authors: Nicolo' Genesio
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ */
+
+#ifndef SIMPLE_INC
+#define SIMPLE_INC
```

Comment:
See https://github.com/robotology/yarp/blob/master/.github/CONTRIBUTING.md#header-guards for a guide on how to name header guards.